packages/fiori-elements-writer/src/data/annotationReuseLibs.ts

```diff
@@ -0,0 +1,58 @@
+import { OdataVersion } from '../types';
+
+interface AnnotationReuseLibsEntry {
+    annotation: string;
+    reuseLib: string;
+}
+
+type AnnotationReuseLibs = {
+    [V in OdataVersion]: [AnnotationReuseLibsEntry] | [];
+};
+
+export const annotationReuseLibs: AnnotationReuseLibs = {
+    [OdataVersion.v2]: [],
+    [OdataVersion.v4]: [
+        {
+            annotation: 'UI.Note',
+            reuseLib: 'sap.nw.core.gbt.notes.lib.reuse'
+        }
+    ]
+};
+
+/**
+ * Gets the reuse libraries associated with annotation entries in meatadata
+ *
+ * @param version - The odata service version reuse library supports
```

Comment:
```suggestion
 * @param version - the odata service version 
```

The current description is a bit confusing